deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -251,7 +251,7 @@ class ChatViewController: MessagesViewController {
 
         if disableWriting {
             menuItems = [
-                UIMenuItem(title: String.localized("start_chat"), action: #selector(MessageCollectionViewCell.messageStartChat(_:))),
+                UIMenuItem(title: String.localized("welcome_start_messaging"), action: #selector(MessageCollectionViewCell.messageStartChat(_:))),
```

Comment:
no, welcome_start_messaging is the wrong string here. this string is used on the inital welcome string.